hammett/stopwatch/stats.py

```diff
@@ -0,0 +1,66 @@
+"""The module contains classes that implement the processing of handler statistics."""
+
+import json
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+from hammett.core.handlers import get_handler_name
+
+if TYPE_CHECKING:
+    from hammett.core.constants import HandlerStats
+    from hammett.types import Handler
+
+
+class BaseStatsProcessor:
+    """Base class for implementing handler statistics processing."""
+
+    def __init__(self, handler: 'Handler', stats: 'HandlerStats') -> None:
+        """Save `handler` and `stats` values to the relevant attributes."""
+        self.handler = handler
+        self.stats = stats
+
+    @classmethod
+    async def on_exit(cls) -> None:
+        """Run after application is stopped."""
+
+    async def process(self) -> None:
+        """Run after handler is called and statistics are retrieved."""
+        raise NotImplementedError
+
+
+class PrintStatsProcessor(BaseStatsProcessor):
+    """The class implements the output of the handler stats via print."""
+
+    async def process(self) -> None:
+        """Print handler stats."""
+        msg = (
+            f'{get_handler_name(self.handler)}:\n'
+            f'  CPU time: {self.stats["cpu_time"]}\n'
+            f'  Select time: {self.stats["select_time"]}\n'
+            f'  Other IO time: {self.stats["other_io_time"]}\n'
+            f'  Real time: {self.stats["real_time"]}\n'
+        )
+        print(msg)  # noqa: T201
+
+
+class JsonStatsProcessor(BaseStatsProcessor):
+    """The class collects the stats of all handlers and dumps them into a json file."""
```

Comment:
`The class collects the stats from all handlers and dumps them into a JSON file.`